src/tabs/presets/FavoritePresets.js

```diff
@@ -0,0 +1,99 @@
+
+const s_maxFavoritePresetsCount = 50;
+const s_favoritePresetsListConfigStorageName = "FavoritePresetsList";
+
+class FavoritePreset {
+    constructor(presetPath){
+        this.presetPath = presetPath;
+        this.lastPickDate = Date.now();
+    }
+}
+
+
+class FavoritePresetsData {
+    constructor() {
+        this._favoritePresetsList = [];
+    }
+
+    _sort() {
+        this._favoritePresetsList.sort((a, b) => (a.lastPickDate - b.lastPickDate));
```

Comment:
```suggestion
        this._favoritePresetsList.sort(a, b => a.lastPickDate - b.lastPickDate);
```
Please remove redundant parenthesis.

Reply:
that won't run and harder to read

Reply:
Okay strange. Approved.